src/library/internal_link.rs

```diff
@@ -1,6 +1,4 @@
-use crate::library::toc::{ToC, TocItem, ParseError};
-use config::Package;
-use fpm::Package;
+use crate::library::toc::{TocItem, ParseError};
```

Comment:
Remove all the `use` statements and use full paths.

Reply:
Resolved it.